src/app/components/mno-data-sync/mno-data-sync.directive.coffee

```diff
@@ -0,0 +1,16 @@
+
+angular.module 'mnoEnterpriseAngular'
+  .directive('mnoDataSync', ->
+    return {
+      restrict: 'EA'
+      templateUrl: 'app/components/mno-data-sync/mno-data-sync.html',
+      scope: {
+        appsData: '='
```

Comment:
I would conflict them with the response of:

        MnoeMarketplace.getApps().then(
          (response) ->
            response = response.plain()
            $scope.categories = response.categories
            $scope.apps = response.apps
        )